pom.xml

```diff
@@ -26,6 +26,12 @@
             <artifactId>mockito-core</artifactId>
             <version>1.8.5</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.hamcrest</groupId>
+                    <artifactId>hamcrest-core</artifactId>
```

Comment:
Why troublesome?

Just picking up the version from the test harness rather than Mockito.

Probably better would be to have the parent POM include the Enforcer rule to use the newest version of a transitive dependency rather than necessarily the closest. Have suggested it before. CC @andresrc